rails_app/.rubocop.yml

```diff
@@ -0,0 +1,66 @@
+# The guiding principle of classes is SRP, SRP can't be accurately measured by LoC
+Metrics/ClassLength:
+  Max: 1500
+
+Metrics/MethodLength:
+  Max: 20
+
+Metrics/LineLength:
+  Max: 120
+
+Metrics/AbcSize:
+  Max: 17
+
+Rails:
+  Enabled: true
+Rails/Delegate:
+  Enabled: false
+Rails/HttpPositionalArguments:
+  Enabled: false
+
+# Most readable form.
+Style/AlignHash:
```

Comment:
Could not find these in ![docs](https://rubocop.readthedocs.io/en/latest/cops_style/)

Reply:
http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/AlignHash

Reply:
so they suck about keeping their docs huh, sorry for that